vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItemFactory.kt

```diff
@@ -207,7 +207,9 @@ class RoomSummaryItemFactory @Inject constructor(
 
     private fun getSearchResultSubtitle(roomSummary: RoomSummary): String {
         val userId = roomSummary.directUserId
-        val spaceName = roomSummary.spaceParents?.firstOrNull()?.roomSummary?.name
+        val spaceName = roomSummary.flattenParents
+                .takeIf { it.isNotEmpty() }
+                ?.joinToString(", ") { it.name }
```

Comment:
Although this is nice, I'm thinking now to maintain parity with web, we only take the last element in `flattenParents` for the subtitle. If design would deem it's better to have all parents separated by comma, we can have a meta ticket to change that across all platforms

Reply:
Doesn't look to me what web is taking the last one only. What are the requirement exactly?
Can you take over it as I believe you are the one that initialy developed it?